spec/unit/lib/cloud_controller/paging/pagination_options_spec.rb

```diff
@@ -115,31 +115,66 @@ module VCAP::CloudController
       end
 
       it 'does not add default values when valid options are specified' do
-        expect(pagination_options.page).to eq(page)
-        expect(pagination_options.per_page).to eq(per_page)
+        expect(pagination_options.page).to eq(page.to_i)
+        expect(pagination_options.per_page).to eq(per_page.to_i)
         expect(pagination_options.order_by).to eq(order_by)
         expect(pagination_options.order_direction).to eq(order_direction)
       end
     end
 
     describe 'validations' do
       context 'when page is not an integer' do
-        let(:params) { { page: 3.5 } }
+        let(:params) { { page: '3.5' } }
```

Comment:
what's the behavior for a case like `page: '3.0'` ?

Reply:
Looks like this is invalid too which is a bit unfortunate but seems to be the behaviour of the `only_integer` flag to the validator